src/main/java/de/retest/persistence/migration/XmlTransformer.java

```diff
@@ -57,19 +55,13 @@ public final InputStream transform( final InputStream inputStream ) {
 				final XMLEvent nextEvent = eventReader.nextEvent();
 				convert( nextEvent, eventWriter );
 			}
-
 			eventReader.close();
 			eventWriter.flush();
 			eventWriter.close();
-			out.close();
 
 			return new LZ4BlockInputStream( new DeleteOnCloseFileInputStream( tmpFile ) );
-		} catch ( final XMLStreamException e ) {
-			throw new RuntimeException( e );
-		} catch ( final FactoryConfigurationError e ) {
+		} catch ( final XMLStreamException | FactoryConfigurationError | IOException e ) {
```

Comment:
Why do you catch IOException, if you specify in the method that it is allowed to throw?